<launch>

    <param name="num_robots" value="1" type="int"/>

    <arg name="uri1" default="radio://0/80/2M/E7E7E7E7E7" />
  
    <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch" />
  
    <group ns="cf1">
  	
  	  <!--<param name="crazyflie/pid_rate/yaw_kp" value="200"/>-->
  	
  	  <node pkg="crazyflie_driver" type="crazyflie_add" name="crazyflie_add" output="screen">
  	  		<param name="uri" value="$(arg uri1)" />
  	    	<param name="tf_prefix" value="cf1" />
  	    	<param name="enable_logging" value="False" />
  	
  	    	<!--
  	    	  <rosparam>
  	    	    genericLogTopics: ["log1"]
  	    	    genericLogTopicFrequencies: [10]
  	    	    genericLogTopic_log1_Variables: ["ctrltarget.yaw", "stabilizer.yaw"]
  	    	  </rosparam> 
  	    	-->
  	
  	  </node>
  	
  	  <!-- <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot_x" args="log1/values[0] log1/values[1]"/> -->
  	
  	</group>


	<!--
    <include file="$(arg launch_path)/modquad.launch">
        <arg name="model" value="modquad01"/>

        <arg name="init_x" default="0.0"/>
        <arg name="init_y" default="0.0"/>


        <arg name="color_r" default="0.0"/>
        <arg name="color_g" default="1.0"/>
        <arg name="color_b" default="0.0"/>
    </include>
	-->

    <!--Robot Visualization-->
    <!--<node pkg="modquad_simulator" type="visualizer.py" name="visual"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find modquad_simulator)/rviz/thirteen-robots.rviz" required="true"/>
	-->


    <!--<node pkg="dock_manager" type="docking_detector.py" name="docking_det"/>-->
    <!--<node pkg="modquad_simulator" type="docking_sim.py" name="docking_sim"/>-->
    <!--<node pkg="dock_manager" type="odom_multiplexer.py" name="odom_mux"/>-->


    <node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster"
          args="1 0 0 0 0 0 1 world link1"/>

	<node name="vicon_test" pkg="modquad_simulator" type="vicon_test.py" output="screen">
	</node>

</launch>
